docs/api.rst

```diff
@@ -837,7 +888,7 @@ takeWhile
 
   takeWhile :: (a -> bool) -> Stream a -> Stream a
 
-Keep all events until predicate returns false, and discard the rest.::
+Keep all events until predicate returns ``false``, and discard the rest. ::
```

Comment:
:+1: I like that you've made this explicitly boolean `false` (rather than "falsy"), to reinforce the strong typing mind-set.  How about we do the same above with `skipRepeatsWith`?

Reply:
Done!